net-utils/src/ip_echo_server.rs

```diff
@@ -18,6 +19,15 @@ use {
 
 pub type IpEchoServer = Runtime;
 
+// Enforce a minimum of two threads:
+// - One thread to monitor the TcpListener and spawn async tasks
+// - One thread to service the spawned tasks
+pub const MINIMUM_IP_ECHO_SERVER_THREADS: usize = 2;
+// IP echo requests require little computation and come in fairly infrequently,
+// so keep the number of server workers small to avoid overhead
+pub const DEFAULT_IP_ECHO_SERVER_THREADS: usize = MINIMUM_IP_ECHO_SERVER_THREADS;
```

Comment:
general question: why do you create two consts here instead of just one and using that everywhere?

Reply:
Despite being equal in value, each constant has a slightly different meaning IMO:
- The minimum is the absolute minimum number of threads that we should allocate for the server. Any number less than this could be problematic (0 will panic and 1 will mean that we can't sit on the TcpListener while we're servicing a request)
- The default is the recommended number of threads for this server, which is backed up by the data I provided in the description.

In this particular case, I'll agree it looks redundant given that they're equal. But, the min is based on the design of the server which shouldn't change unless we re-architect it. Whereas the default might change if something changes in the future (maybe we start using these echo requests ALOT, like 10's per second)

Reply:
i see. two knobs that are currently equal but default could change in the future. makes sense. thank you!